# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1206649085 14400
#      Thu Mar 27 16:18:05 2008 -0400
# Node ID f3493c40a0bdf9b21378b4d8a2ca55575010501d
# Parent  5f6e11567f7065a763de2895f3cfac49ca4c9863
__convn__.cc: fix typo in previous change

diff --git a/src/DLD-FUNCTIONS/__convn__.cc b/src/DLD-FUNCTIONS/__convn__.cc
--- a/src/DLD-FUNCTIONS/__convn__.cc
+++ b/src/DLD-FUNCTIONS/__convn__.cc
@@ -169,15 +169,16 @@ Undocumented internal function.\n\
 	      const NDArray b = args (1).array_value ();
 
 	      if (! error_state)
 		retval = convn (a, b);
 	    }
 	  else
 	    error ("__convn__: invalid call");
 	}
-      error ("__convn__: invalid call");
+      else
+	error ("__convn__: invalid call");
     }
   else
     print_usage ();
     
   return retval;
 }
